apps/api/src/app/change/e2e/get-changes.e2e.ts

```diff
@@ -58,6 +58,7 @@ describe('Get changes', () => {
       description: testTemplate.description,
       steps: [],
       notificationGroupId: session.notificationGroups[0]._id,
+      identifier: undefined,
```

Comment:
Look at might other comment on how we can skip the undefined assignments by making the property optional in Ts.